spec/datetimeformat.html

```diff
@@ -73,29 +73,11 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. Set _dateTimeFormat_.[[Locale]] to _r_.[[locale]].
         1. Let _calendar_ be _r_.[[ca]].
         1. Set _dateTimeFormat_.[[Calendar]] to _calendar_.
-        1. Set _dateTimeFormat_.[[HourCycle]] to _r_.[[hc]].
         1. Set _dateTimeFormat_.[[NumberingSystem]] to _r_.[[nu]].
         1. Let _dataLocale_ be _r_.[[dataLocale]].
-        1. Let _timeZone_ be ? Get(_options_, *"timeZone"*).
-        1. If _timeZone_ is *undefined*, then
-          1. Let _timeZone_ be DefaultTimeZone().
-        1. Else,
-          1. Let _timeZone_ be ? ToString(_timeZone_).
-          1. If the result of IsValidTimeZoneName(_timeZone_) is *false*, then
-            1. Throw a *RangeError* exception.
-          1. Let _timeZone_ be CanonicalizeTimeZoneName(_timeZone_).
-        1. Set _dateTimeFormat_.[[TimeZone]] to _timeZone_.
-        1. Let _formatOptions_ be a new Record.
-        1. For each row of <emu-xref href="#table-datetimeformat-components"></emu-xref>, except the header row, in table order, do
-          1. Let _prop_ be the name given in the Property column of the row.
-          1. If _prop_ is *"fractionalSecondDigits"*, then
-            1. Let _value_ be ? GetNumberOption(_options_, *"fractionalSecondDigits"*, 1, 3, *undefined*).
-          1. Else,
-            1. Let _value_ be ? GetOption(_options_, _prop_, *"string"*, &laquo; the strings given in the Values column of the row &raquo;, *undefined*).
-          1. Set _formatOptions_.[[&lt;_prop_&gt;]] to _value_.
         1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
         1. Let _hcDefault_ be _dataLocaleData_.[[hourCycle]].
-        1. Let _hc_ be _dateTimeFormat_.[[HourCycle]].
+        1. Let _hc_ be _r_.[[hc]].
         1. If _hc_ is *null*, then
           1. Set _hc_ to _hcDefault_.
```

Comment:
This is still setting _hc_ too many times for my taste, but that can be addressed later. For the record, I'd prefer something like
```
        1. Let _hcDefault_ be _dataLocaleData_.[[hourCycle]].
        1. If _hour12_ is *true*, then
          1. If _hcDefault_ is *"h11"* or *"h23"*, let _hc_ be *"h11"*. Otherwise, let _hc_ be *"h12"*.
        1. Else if _hour12_ is *false*, then
          1. If _hcDefault_ is *"h11"* or *"h23"*, let _hc_ be *"h23"*. Otherwise, let _hc_ be *"h24"*.
        1. Else,
          1. Assert: _hour12_ is *undefined*.
          1. Let _hc_ be _r_.[[hc]].
          1. If _hc_ is *null*, set _hc_ to _hcDefault_.
        1. Set _dateTimeFormat_.[[HourCycle]] to _hc_.
```